### DISCUSSION TITLE
Center items after a delete with flex grid ?

### DISCUSSION THREAD

**Discussion by Nico7522** [tailwindlabs/tailwindcss#11304] [General]

**Comment by wongjn** [comment]
For three items, you can center them by having grid columns equal to 3:
```diff
-<div className="sm:grid sm:grid-cols-4 sm:gap-4 sm:justify-center sm:content-center m-auto bg-white w-3/4 max-w-7xl flex flex-col">
+<div className="sm:grid sm:grid-cols-3 sm:gap-4 sm:justify-center sm:content-center m-auto bg-white w-3/4 max-w-7xl flex flex-col">
```
[See live example on Tailwind Play](https://play.tailwindcss.com/LVt9sq9p8c).

**Reply by Nico7522** [reply]
But now if I have 2 items there no more center.

What i need is to always got my items center wathever the number of items I have

**Reply by wongjn** [reply]
You could consider having 8 grid column tracks, having the items span two columns, and then offset the first item as necessary: https://play.tailwindcss.com/v97DchQ7Ks
